### PR TITLE
Issue #3999: Split and Organize Checkstyle inputs by Test for FileLength

### PR DESCRIPTION
**Pull request by johnyleebrown**
Fixes Issue #3999

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/filelength/InputFileLength.java [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+/**
+ * Contains simple mistakes:
+ * - Long lines
+ * - Tabs
+ * - Format of variables and parameters
+ * - Order of modifiers
+ * @author Oliver Burn
+ **/
+final class InputSimple
```
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/filelength/InputFileLength.java]: please change class to `InputFileLength` like file name.

Comment (johnyleebrown) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/filelength/InputFileLength.java]: @rnveach Done.

**Comment by rnveach** [comment]
@johnyleebrown As you are fixing CI, please squash all commits to 1, keep them at 1, and rebase off the latest master.

**Comment by johnyleebrown** [comment]
@rnveach Done.

**Comment by rnveach** [comment]
@johnyleebrown Travis failure is code coverage failure.
Can you generate code coverage report on this branch, https://github.com/checkstyle/checkstyle/wiki/How-to-generate-UT-coverage-report , and see what the problem is.
If it is the check that is missing coverage, please re-verify you didn't remove anything important from the input you copied.

**Comment by johnyleebrown** [comment]
@rnveach generated code coverage report, verified with `mvn verify`.

**Comment by rnveach** [comment]
weird, I refreshed the job and now it passes.

@romani When I run code coverage on current master, I am seeing missing branch coverage for `IllegalInstantiationCheck`. Can you confirm?

**Pull request merged**